Mailmap updates

QuLogic opened the pull request:
I'm pulling this back as far as possible to keep everything in sync; there may be some conflicts when propagating branches forward, but this version is intended to supersede all copies.

For those of you wondering why you are being tagged here, the `.mailmap` file ensures that you will be counted as a single person as part of [matplotlib's GitHub stats](http://matplotlib.org/users/github_stats.html). Please confirm that your name and email (the first entry on a line) are the way you want to be listed. As there are some layout changes to this file in this PR, you may want to check only the last commit which only contains the addition of your names.

- [X] @afvincent
- [x] @basharovV
- [x] @bcongdon
- [x] @danhickstein
- [x] @dsquareindia
- [x] @flothesof
- [ ] @grdlok
- [x] @hamogu
- [x] @JackKelly
- [x] @JGoutin
- [x] @klaragerlei
- [x] @krischer
- [x] @kthyng
- [x] @LindyBalboa
- [x] @maluethi
- [x] ~~@mbyt~~
- [x] @ocefpaf
- [x] @phobson
- [x] @rasbt
- [x] @ryanbelt
- [ ] @smheidrich
- [X] @story645
- [x] @TrishGillett

afvincent commented:
@QuLogic : my entry is fine, thank you.

kthyng commented:
@QuLogic mine is good, thanks!

On Sun, Sep 18, 2016 at 8:20 AM, Adrien F. Vincent <notifications@github.com
> wrote:

> @QuLogic <https://github.com/QuLogic> : my entry is fine, thank you.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/matplotlib/matplotlib/pull/7127#issuecomment-247847309>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADU2Bf9lSR64d26lvrxrmLqZuDHLYroUks5qrTqQgaJpZM4J_zdC>
> .
>



-- 
Kristen M. Thyng
Assistant Research Professor
Department of Oceanography
Texas A&M University
Eller O&M 607
http://kristenthyng.com

rasbt commented:
@QuLogic Thanks for the note. Instead of listing me as "rasbt," it would look nicer if you could list me as Sebastian Raschka. Thanks!

klaragerlei commented:
@QuLogic <https://github.com/QuLogic> This is fine, thank you!

On 18 September 2016 at 06:20, Elliott Sales de Andrade <
notifications@github.com> wrote:

> I'm pulling this back as far as possible to keep everything in sync; there
> may be some conflicts when propagating branches forward, but this version
> is intended to supersede all copies.
>
> For those of you wondering why you are being tagged here, the .mailmap
> file ensures that you will be counted as a single person as part of matplotlib's
> GitHub stats <http://matplotlib.org/users/github_stats.html>. Please
> confirm that your name and email (the first entry on a line) are the way
> you want to be listed. As there are some layout changes to this file in
> this PR, you may want to check only the last commit which only contains the
> addition of your names.
>
>    - @afvincent <https://github.com/afvincent>
>    - @basharovV <https://github.com/basharovV>
>    - @bcongdon <https://github.com/bcongdon>
>    - @danhickstein <https://github.com/danhickstein>
>    - @dsquareindia <https://github.com/dsquareindia>
>    - @flothesof <https://github.com/flothesof>
>    - @grdlok <https://github.com/grdlok>
>    - @hamogu <https://github.com/hamogu>
>    - @JackKelly <https://github.com/JackKelly>
>    - @JGoutin <https://github.com/JGoutin>
>    - @klaragerlei <https://github.com/klaragerlei>
>    - @krischer <https://github.com/krischer>
>    - @kthyng <https://github.com/kthyng>
>    - @maluethi <https://github.com/maluethi>
>    - @mbyt <https://github.com/mbyt>
>    - @ocefpaf <https://github.com/ocefpaf>
>    - @pmhobson
>    - @rasbt <https://github.com/rasbt>
>    - @ryanbelt <https://github.com/ryanbelt>
>    - @smheidrich <https://github.com/smheidrich>
>    - @story645 <https://github.com/story645>
>
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/matplotlib/matplotlib/pull/7127
> Commit Summary
>
>    - Backport .mailmap changes from master.
>    - Split names in .mailmap for easier viewing.
>    - Add some new entries to .mailmap.
>
> File Changes
>
>    - *M* .mailmap
>    <https://github.com/matplotlib/matplotlib/pull/7127/files#diff-0>
>    (194)
>
> Patch Links:
>
>    - https://github.com/matplotlib/matplotlib/pull/7127.patch
>    - https://github.com/matplotlib/matplotlib/pull/7127.diff
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/matplotlib/matplotlib/pull/7127>, or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AP4Nn12SR_y6nTzRdPwcKxznpHy7k1Pcks5qrMoDgaJpZM4J_zdC>
> .
>

DanHickstein commented:
My name could be listed as "Dan Hickstein" rather than just DanHickstein. Thanks!

phobson commented on .mailmap:
```diff
@@ -1,62 +1,176 @@
+Patrick Chen <pat.chen@mail.utoronto.ca>
+
+Paul Hobson <pmhobson@gmail.com>
+Paul Hobson <pmhobson@gmail.com> vagrant <vagrant@precise32.(none)>
```
Not sure what the vagrant part is. Did I push a commit from a VM along time ago?

Otherwise, this looks good to me.

QuLogic replied:
Yes, #2768. It is a rather generic name though, so we'd have to be careful no-one else uses it, or maybe not bother fixing that one commit's attribution.

phobson replied:
Oof. Sorry about that.

I'm ok with that commit not getting attributed to me.

The main thing I'm concerned about is that y'all (the core team) have as few headaches as possible. I'm on board with whatever you chose to do wit this.

mbyt commented on .mailmap:
```diff
@@ -1,62 +1,176 @@
 MinRK <benjaminrk@gmail.com> Min RK <minrk@kerbin.local>
-Nelle Varoquaux <nelle.varoquaux@gmail.com> Varoquaux <nelle.varoquaux@gmail.com>
+
+Moritz Beyreuther <random.seed@web.de>
```
Hi @QuLogic, funny to meet you again in this project `:)` (and not in obspy). If you do not mind, I would prefer the pseudonym mbyt as name instead.. I do it like this everywhere on github... Email is right.

QuLogic replied:
Sure, no problem. You've still got your name in ObsPy commits, but I realize now those're pretty old and all the new ones are this style.

mbyt replied:
Great, advanced thanks!

ocefpaf commented on .mailmap:
```diff
@@ -1,62 +1,176 @@
+Eric Ma <ericmajinglong@gmail.com> <ericmjl@Erics-MacBook-Pro.local>
+Eric Ma <ericmajinglong@gmail.com> <ericmjl@users.noreply.github.com>
+
+Filipe <ocefpaf@gmail.com>
```
Filipe Fernandes <ocefpaf@gmail.com>

(Thanks for doing this @QuLogic.)

bcongdon commented:
@QuLogic My entry's good. Thanks!

maluethi commented on .mailmap:
```diff
@@ -1,62 +1,176 @@
+
+matthias <maluethi@protonmail.ch>
+matthias <maluethi@protonmail.ch> <matthias.luethi@lhep.unibe.ch>
+
```
Matthias Lüthi <maluethi@protonmail.ch> <matthias.luethi@lhep.unibe.ch>

Thanks @QuLogic

krischer commented on .mailmap:
```diff
@@ -1,62 +1,176 @@
+
+Levi Kilcher <levi.kilcher@nrel.gov>
+
+Lion Krischer <krischer@geophysik.uni-muenchen.de>
```
+Lion Krischer <lion.krischer@gmail.com> <krischer@geophysik.uni-muenchen.de>

JackKelly commented on .mailmap:
```diff
@@ -1,62 +1,176 @@
+
+J. Goutin <JGoutin@users.noreply.github.com> JGoutin <ginnungagap@free.fr>
+
+Jack Kelly <jack-list@xlk.org.uk> Jack (aka Daniel) Kelly <daniel.kelly10@imperial.ac.uk>
```
Please could we change my line to:

`Jack Kelly <jack.kelly@imperial.ac.uk>`

Thank you!

JGoutin commented on .mailmap:
```diff
@@ -1,62 +1,176 @@
+Hans Moritz Günther <moritz.guenther@gmx.de>
+
+J. Goutin <JGoutin@users.noreply.github.com> JGoutin <ginnungagap@free.fr>
+
```
Please, could you change my line to:
J.Goutin <JGoutin@users.noreply.github.com>
Thanks

QuLogic replied:
Without the space?

JGoutin replied:
Keep the space.

QuLogic replied:
Not sure I understand what you want different then?

JGoutin replied:
I just want removing the second mail address.

Thanks.

QuLogic replied:
I can leave it out if you don't want them associated, but note the second address already appears in commits alongside your username; this line ensures they're treated as the same person.

JGoutin replied:
This is OK for me. Thanks.

hamogu commented:
Looks good to me.

tacaswell commented:
@QuLogic Feel free to self-merge this.

QuLogic commented:
Updated with corrections from @DanHickstein, @ocefpaf, @JackKelly, @krischer, @maluethi, @mbyt, and @rasbt.

Added @LindyBalboa and @TrishGillett from gitter.

TrishGillett commented:
Mine looks good, thanks!

LindyBalboa commented:
Alright I'm gonna be a jerk and change my mind. After panning through and seeing that most go with the name (and Travis needing restarted anyways), I'll bite. 

Conner R. Phillips 

will work. Keep it simple and match the email I suppose. Thanks Elliott.

basharovV commented:
@QuLogic Please could mine be listed as: Slav Basharov
Email is fine. Thanks!

ryanbelt commented:
@QuLogic mine is good to go, thank you

QuLogic commented:
At the end of the month, I think I will merge this. We're at >80% response and many of the remaining haven't even been on GitHub in a while (publicly, at least.)

flothesof commented:
Could my name be listed as Florian Le Bourdais? 
Thanks a lot @QuLogic !

Regards,
Florian

QuLogic commented:
Hopefully last update with corrections from @LindyBalboa, @basharovV, and @flothesof.

dsquareindia commented:
@QuLogic mine is also fine. Thanks!

mbyt commented:
@QuLogic not sure if I understand that here correctly. From the conversation above I assumed the `.mailmap` entry would be `mbyt <random.seed@web.de>`.

Now, for me it looks like in https://github.com/matplotlib/matplotlib/pull/7127/commits/08372609f6e1b0d313dfbcd487d330c5b45c5413 the entry was deleted completely from the list...

QuLogic commented:
@mbyt Your commits are all correct; there's not really any need for your entry here.
